### ISSUE TITLE
firstHalf: wrong solution

### ISSUE THREAD

**Issue by HamadAlsayed** [the-winter/codingjs#33]
firstHalf solution is the full string instead of half of it.

#24 was supposed to fix this but it looks like the bug is still present

**Comment by HamadAlsayed** [comment]
<img width="554" alt="Screenshot 2023-10-02 at 12 03 29" src="https://github.com/the-winter/codingjs/assets/24865583/6b3b85ab-a82d-44e6-bf71-45dfd5236399">

**Comment by HamadAlsayed** [comment]
I also realize it's supposed to use only even strings and 'abcdefg' is not even. It's comprised of 7 characters.

**Comment by usernotfound8** [comment]
> I also realize it's supposed to use only even strings and 'abcdefg' is not even. It's comprised of 7 characters.

I'd imagine for that you'd just return the string
Also almost 4 months later and it's still not fixed
![Screenshot 2024-01-24 15 02 05](https://github.com/the-winter/codingjs/assets/144141314/2a50ff1d-46da-4aa6-bf52-f67d64681734)

**Comment by HamadAlsayed** [comment]
I mean it seems the original authors of this have disappeared. None of the pull requests have been approved in long years.

**Comment by usernotfound8** [comment]
> I mean it seems the original authors of this have disappeared. None of the pull requests have been approved in long years.

Yeah, it's kinda sad. I guess other people could make a new website but I don't think people would want to just to fix a couple issues.
